src/drivers/mct/bld/testdir/user_nl_cpl

```diff
@@ -0,0 +1,3 @@
+&a
+  drydep_method = 'xactive_lnd'
```

Comment:
I don't think that we want a testdir directory and if we do it should probably be located in scripts/tests